src/webui/btjson.cpp

```diff
@@ -225,32 +225,36 @@ static const char KEY_LOG_PEER_IP[] = "ip";
 static const char KEY_LOG_PEER_BLOCKED[] = "blocked";
 static const char KEY_LOG_PEER_REASON[] = "reason";
 
-QVariantMap getTranserInfoMap();
-QVariantMap toMap(BitTorrent::TorrentHandle *const torrent);
-void processMap(QVariantMap prevData, QVariantMap data, QVariantMap &syncData);
-void processHash(QVariantHash prevData, QVariantHash data, QVariantMap &syncData, QVariantList &removedItems);
-void processList(QVariantList prevData, QVariantList data, QVariantList &syncData, QVariantList &removedItems);
-QVariantMap generateSyncData(int acceptedResponseId, QVariantMap data, QVariantMap &lastAcceptedData,  QVariantMap &lastData);
-
-class QTorrentCompare
+static QVariantMap getTranserInfoMap();
+static QVariantMap toMap(BitTorrent::TorrentHandle *const torrent);
+static void processMap(const QVariantMap &prevData, const QVariantMap &data, QVariantMap &syncData);
+static void processHash(QVariantHash prevData, const QVariantHash &data, QVariantMap &syncData, QVariantList &removedItems);
+static void processList(QVariantList prevData, const QVariantList &data, QVariantList &syncData, QVariantList &removedItems);
+static QVariantMap generateSyncData(int acceptedResponseId, const QVariantMap &data, QVariantMap &lastAcceptedData, QVariantMap &lastData);
```

Comment:
>Right, AFAIK that requires moving a lot of code (function body) to the top of the file (to the anonymous namespace), is that fine?

I misunderstood you. What I meant: Move both the declarations and definitions inside the namespace. But keep separate.